modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -16,6 +16,7 @@ import type {
   PopupInfo,
   FieldInfo
 } from '@loaders.gl/i3s';
+import type {GLTF_EXT_feature_metadata} from 'modules/gltf/src/lib/types/gltf-json-schema';
```

Comment:
This type of explicit path imports will break in production. Let's avoid it all costs.

Reply:
Done